app/pages/adduser/ui.test.js

```diff
@@ -161,9 +161,29 @@ test('should render add user button', async (t) => {
     .expect(addUserButton.exists).ok()
     .expect(await extractInnerText(addUserButton)).eql(content.addUserButtonText)
     .expect(addUserButton.hasClass('nhsuk-u-margin-bottom-9')).ok();
-  // TODO: Uncomment when API allows users to be added
-  // .expect(addUserButton.getAttribute('href')).eql(`/organisations/${organisationDetails.organisationId}/adduser`);
-  // .click()
-  // Add tests for confirmation page for success
+});
+
+test('should navigate to confirmation page when form is filled out and addUser button is clicked', async (t) => {
+  nock(organisationsApiLocalhost)
+    .post('/api/v1/Organisations/org1/Users')
+    .reply(200);
+
+  await pageSetup(t, true);
+  await t.navigateTo('http://localhost:1234/organisations/org1/adduser');
+
+  const firstName = Selector('[data-test-id="question-firstName"] input');
+  const lastName = Selector('[data-test-id="question-lastName"] input');
+  const phoneNumber = Selector('[data-test-id="question-phoneNumber"] input');
+  const emailAddress = Selector('[data-test-id="question-emailAddress"] input');
+  const addUserButton = Selector('[data-test-id="add-user-button"] button');
+
+  await t
+    .expect(addUserButton.exists).ok()
+    .typeText(firstName, 'Peter')
+    .typeText(lastName, 'Parker')
+    .typeText(phoneNumber, '07777777777')
+    .typeText(emailAddress, 'Peter@Parker.com')
```

Comment:
Upto you but we don't really need to type the text into each field. We don't do any validation on the node side. So it being empty is also perfectly fine.

Reply:
it looks nice when it types it in 😄

Reply:
I remember now - I wondered why I couldn't find an example haha